completions/fish/crio.fish

```diff
@@ -105,6 +105,8 @@ complete -c crio -n '__fish_crio_no_subcommand' -f -l metrics-collectors -r -d '
 complete -c crio -n '__fish_crio_no_subcommand' -f -l metrics-key -r -d 'Certificate key for the secure metrics endpoint'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l metrics-port -r -d 'Port for the metrics endpoint'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l metrics-socket -r -d 'Socket for the metrics endpoint'
+complete -c crio -n '__fish_crio_no_subcommand' -f -l minimum-mappable-gid -r -d 'Specify the lowest host GID which can be specified in mappings for a pod that will be run as a UID other than 0'
```

Comment:
I think?

```suggestion
complete -c crio -n '__fish_crio_no_subcommand' -f -l minimum-mappable-gid -r -d 'Specify the lowest host GID which can be specified in mappings for a pod that will be run as a GID other than 0'
```

Reply:
The intent is that pods which weren't already going to be run as root (UID 0) won't be allowed to specify mappings that use the host's root user's UID or primary GID.